Mathlib/Order/Interval/Set/SuccPred.lean

```diff
@@ -66,8 +66,8 @@ lemma insert_Icc_succ_left_eq_Icc (h : a ≤ b) : insert a (Icc (succ a) b) = Ic
   ext x; simp [or_and_left, eq_comm, ← le_iff_eq_or_succ_le]; aesop
 
 lemma insert_Icc_right_eq_Icc_succ (h : a ≤ succ b) :
```

Comment:
The lemmas in this file also seem like they could be aliases of some flipped versions that could exist in `Mathlib.Order.SuccPred.Basic`?

Reply:
Things should happen the other way around. The lemmas in `Order.SuccPred.Basic` should be removed in favor of the ones here. Please revert these changes from this PR